net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/forms/dialogEditWireguardClient.xml

```diff
@@ -24,12 +24,12 @@
         <help>Shared secret (PSK) for this peer.</help>
     </field>
     <field>
-        <id>client.tunneladdress</id>
-        <label>Tunnel Address</label>
+        <id>client.allowedips</id>
```

Comment:
I guess we really cannot change this internal node name without breaking stuff

Reply:
Yes but i think that changing it right now it's the best moment to do before it become stable.
I don't know but maybe most users that are currently using it it's just for testing or personal usage mainly because it's still "unreliable" and in development.
I'm not sure but could be that it just need to reinstall the plugin and reconfigure it to make it work again after the changes?

Reply:
No, we have to keep tunneladdress as ID and only change the label. A reinstall/update would lead to empty values since these doesn't exist. Normally it's ok when switching from devel to stable, but as there's a huge number of users already using it in production I don't want to make them angry :)

Reply:
Ah ok, didn't know was already used in production.

Reply:
Do i need to revert just the id or even the label?